server/server/server.go

```diff
@@ -210,7 +222,9 @@ func (s *Server) Shutdown() error {
 		s.cleanUpSessionsTask.Cancel()
 	}
 
-	s.telemetry.Shutdown()
+	if err := s.telemetry.Shutdown(); err != nil {
```

Comment:
Now if it fails to Shutdown the telemetry is no longer trying to shutdown the store. I think is more correct to log an error there instead of returning the error. Maybe we can use a multi-error here.